packages/cli/index.js

```diff
@@ -318,16 +318,38 @@ program
       .catch(handleError);
   });
 
+program
+  .command('archive [versionName] [archivePath]')
+  .alias('ar')
+  .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .description('archive a version of the site, which is not affected by later changes to the site')
+  .action((versionName, userSpecifiedArchivePath, options) => {
+    if (!versionName) {
+      logger.error('Please specify a name for the archived version.');
+    }
+    const archivePath = userSpecifiedArchivePath || 'version';
+    const rootFolder = path.resolve(process.cwd());
+    const outputFolder = path.join(rootFolder, archivePath, versionName);
+    new Site(rootFolder, outputFolder, undefined, undefined, options.siteConfig)
+      .archive(versionName, archivePath)
+      .then(() => {
+        logger.info(`Successfully archived ${versionName} at ${archivePath}/${versionName}`);
+      })
+      .catch(handleError);
+  });
+
 program
   .command('deploy')
   .alias('d')
   .description('deploy the site to the repo\'s Github pages')
   .option('-c, --ci [githubTokenName]', 'deploy the site in CI Environment [GITHUB_TOKEN]')
   .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .option('-v, --versions <versionFolder>', 'specify the path where versions are stored')
   .action((options) => {
     const rootFolder = path.resolve(process.cwd());
     const outputRoot = path.join(rootFolder, '_site');
-    new Site(rootFolder, outputRoot, undefined, undefined, options.siteConfig).deploy(options.ci)
+    const site = new Site(rootFolder, outputRoot, undefined, undefined, options.siteConfig);
+    site.deploy(options.ci)
```

Comment:
Not sure why this line is edited

Reply:
Ended up reverting the changes related to this, and forgot to change it back :')